builders/ethereum/precompiles/utility/non-specific.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Non-Network Specific Precompiles
+description: Learn how to use precompiled contracts, which are not specific to Ethereum or Moonbeam, yet are supported for use in your application.
+keywords: ethereum, moonbeam, StorageCleaner, ECRecoverPublicKey
```

Comment:
Just so you know, we started to apply keywords on some pages, and then realized that it wasn't worth the effort. Keywords don't do much anymore in terms of SEO. So you don't need to worry about these in the future, they're fine here though